Resources/Prototypes/_White/Entities/Objects/Weapons/Guns/Turrets/ship_turrets.yml

```
@@ -0,0 +1,174 @@
+  components:
+  - type: Sprite
+    state: security
+  - type: MachineBoard
+    prototype: WeaponTurretShipLight
+    requirements:
+      MatterBin: 1
+      Manipulator: 2
+      Capacitor: 3
+    materialRequirements:
+      Cable: 5
+      Steel: 10 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)